packages/common/src/lib/common/tools.ts

```diff
@@ -1309,13 +1354,11 @@ async function getRepositoryFileAsync(url, hash, force, _actualRepo) {
 /**
  * Sends the given object to the diagnosis server
  *
- * @param {object} obj - diagnosis object
- * @return {Promise<void>}
+ * @param obj - diagnosis object
  */
-async function sendDiagInfo(obj) {
+async function sendDiagInfo(obj: Record<string, any>): Promise<void> {
```

Comment:
Also we should define how the object actually looks like.

Reply:
currently only used in main.js and main.js can put into it what it wants, just needs to be an Object so we can handle the data type